script.js

```diff
@@ -14,3 +70,490 @@ backMenu.forEach((element) => element.addEventListener('click', () => {
+          <li><a href="https://medium.com/BlessmanNewton"><i class="fa fa-medium" aria-hidden="true"></i></a></li>
+      </ul>
+  </div>
+  <hr class="hr_footer">
+  `;
+
+  body.appendChild(footer);
+  // ----contact section above -----
+
+  const view = document.querySelectorAll('.view-project');
+  view.forEach((element) => element.addEventListener('click', () => {
+    const show = document.createElement('div');
+    const detailPop = document.createElement('div');
+    show.className = 'show';
+    detailPop.className = 'pop';
+    detailPop.innerHTML = `
+                      <div class="popup-header">
+                        <h2 class="desktop">Keeping track of hundreds
+                         of components website</h2>
+                         <h2 class="mob-title">Multi Post Stories</h2>
+                        <span id="close">&times;</span>
+                      </div>
+                        <div class="popup-button">
+                          <button id="lists">HTML</button>
+                          <button id="lists">Boostrap</button>
+                          <button id="lists">Ruby on rails</button>
+                        </div>
+                      <div class="popup-body">
+                        <img class="desktop" src="${portfolioData[0].image[5]}" alt="">
+                        <img class="mob" src="${portfolioData[0].image[6]}" alt="">
+                        <div class="pop-details">
+                          <p>
+                          Lorem Ipsum is simply dummy text of the
+                          printing and typesetting industry.
+                          Lorem Ipsum has been the industry's
+                          standard dummy text ever since the 1500s,
+                          when an unknown printer took a
+                           galley of type and scrambled it 1960s.
+                          </p>
+
+                          <p class="desktop">
+                          Lorem Ipsum is simply dummy text of the
+                          printing and typesetting industry.
+                          Lorem Ipsum has been the industry's
+                          standard dummy text ever since the 1500s,
+                          when an unknown printer took a
+                           galley of type and scrambled it 1960s.
+                          </p>
+                         
+                          <div class="popup-button">
+                            <button class="popup-btn"><a href="${workData[0].live_see}">See live
+                            <i class="fa fa-external-link" aria-hidden="true"></i></a></button>
+                            <button class="popup-btn git"><a href="${workData[0].link}">See Source
+                                <i class="fa fa-github" aria-hidden="true">
+                                </i></a>
+                            </button>
```

Comment:
> - [ ] As highlighted by the previous code reviewer, it would be appreciated if you could make adjustments to the spacing between the button to align it with the design on Figma. Additionally, please ensure that the icons used match the ones in the original design to maintain consistency. Your attention to these details will contribute to a polished and cohesive user interface, aligning the implementation with the intended design vision.😊